examples/rp_pico/interrupt/main.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022, Nikolay Semenov
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+
+int
+main()
+{
+	Board::initialize();
+
+	using Led = Board::LedGreen;
+	Led::setOutput();
+	Led::set();
+
+	GpioIntHandler::enable();
+
+	// Powers on the LED on the low->high transition, and off on high->low.
+	GpioInput0::setInput();
+	GpioIntHandler::connect<GpioInput0>(Gpio::InputTrigger::BothEdges,
+										[](Gpio::InputTrigger triggers) {
+											Led::set(!!(triggers & Gpio::InputTrigger::RisingEdge));
```

Comment:
```suggestion
											Led::set(triggers & Gpio::InputTrigger::RisingEdge);
```
casts happens automatically because bool arg type

Reply:
main.cpp:29:62: error: no matching function for call to 'modm::platform::GpioStatic<modm::platform::detail::Data25>::set(modm::FlagsOperators<modm::platform::Gpio::InputTrigger, long unsigned int>)'
   29 |            Led::set(triggers & Gpio::InputTrigger::RisingEdge);

Reply:
Thank you for taking the time to review the PR!